compile/enum.go

```diff
@@ -52,19 +54,34 @@ func compileEnum(file string, src *ast.Enum) (*EnumSpec, error) {
 				Reason: err,
 			}
 		}
-
 		value := prev + 1
 		if astItem.Value != nil {
 			value = *astItem.Value
 		}
 		prev = value
 
+		itemAnnotations, err := compileAnnotations(astItem.Annotations)
+		if err != nil {
+			return nil, compileError{
+				Target: astItem.Name,
```

Comment:
This should be `src.Name + "." + astItem.Name` so that we have the enum name in addition to the item name.